packages/internal/tsconfig.json

```diff
@@ -6,5 +6,5 @@
     "tsBuildInfoFile": "dist/tsconfig.tsbuildinfo",
     "outDir": "dist",
   },
-  "include": ["src", "./ambient.d.ts"],
+  "include": ["src/**/*", "./ambient.d.ts", "src/../package.json"],
```

Comment:
Not important - but we ran into issues on the gql/api refactor with this one. What do you think requiring it instead? I did something similar here: [`a5137b8` (#3286)](https://github.com/redwoodjs/redwood/pull/3286/commits/a5137b810629bc9017e789c40d5ec72d0d55495e)

Reply:
Ah dawg yes. I struggled with this a ton last night. So annoying.

Reply:
@dac09 I think we need to reconsider using require, since it's not supported in es modules.